### DISCUSSION TITLE
DBT upstream

### DISCUSSION THREAD

**Discussion by QishunX** [dagster-io/dagster#19516] [Q&A] [answered]
Hi, guys. My dbt project model depends on one asset of dagster, which will generate multi tables in my databse. So in my schema.yml file, do I have do define the meta for each table? Or there's a way that I can define the meta for a list of table?
Something like this:
      - name: a
        meta:
          dagster:
            asset_key: [ "a" ]
      - name: b
        meta:
          dagster:
            asset_key: [ "a" ]
      - name: c
        meta:
          dagster:
            asset_key: [ "a" ]
            
  To be:
meta:
          dagster:
            asset_key: [ "a" ]
-name:a
-name:b
-name:c
      
      
Thanks!

**Comment by rexledesma** [comment, marked as answer]
You have to define the `meta:` for each table. Each table needs a unique `AssetKey`, so you cannot apply the same asset key to all those tables.

See our documentation: https://docs.dagster.io/integrations/dbt/reference#upstream-dependencies.
